src/crypto.ts

```diff
@@ -8,35 +8,35 @@ export type PublicKey = pki.ed25519.NativeBuffer;
 export type SecretKey = pki.ed25519.NativeBuffer;
 export type Signature = pki.ed25519.NativeBuffer;
 
-// NewHash returns a blake2b 256bit hasher.
-function NewHash() {
+// Returns a blake2b 256bit hasher.
+function newHash() {
   return blake.blake2bInit(32, null);
 }
 
-// HashAll takes all given arguments and hashes them.
-export function HashAll(...args: any[]): Uint8Array {
-  const h = NewHash();
+// Takes all given arguments and hashes them.
+export function hashAll(...args: any[]): Uint8Array {
+  const h = newHash();
   for (let i = 0; i < args.length; i++) {
     blake.blake2bUpdate(h, args[i]);
   }
   return blake.blake2bFinal(h);
```

Comment:
- we can remove `newHash` function completely and replace it here with the implementation - it's just used here once, not exported and the blake2 implementation is not being abstracted so there is no need to create a separate function for it
- try to use meaningful variable names `h` -> `hash`

```suggestion
  const hash = blake.blake2bInit(32, null);
  for (let i = 0; i < args.length; i++) {
    blake.blake2bUpdate(hash, args[i]);
  }
  return blake.blake2bFinal(hash);
```

Reply:
This is basically a copy/paste from Sia. I'll keep `newHash` for now since I may use it when implementing encryption.